devops/actions/e2e-tests-win/action.yml

```diff
@@ -35,6 +35,7 @@ runs:
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
       cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLEVEL_ZERO_LIBS_DIR="D:\github\level-zero_win-sdk\lib" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
+      sycl-ls --verbose
```

Comment:
Can we do

```
      echo "::group::sycl-ls --verbose"
      sycl-ls --verbose
      echo "::endgroup::
```

here? Not sure if that works on Windows, though.

Reply:
Yeah, also not sure, will try